### ISSUE TITLE
Infinite "scheduleNewsletter" loop

### ISSUE THREAD

**Issue by brendanlong** [TelescopeJS/Telescope#1585]
If I get a clean repo and run `npm install && npm start`, I get 100% CPU utilization and tons of this:

```
I20170312-12:55:51.143(-4) (percolatestudio_synced-cron.js:84) SyncedCron: Starting "scheduleNewsletter".
I20170312-12:55:51.143(-4) (percolatestudio_synced-cron.js:84) SyncedCron: Finished "scheduleNewsletter".
I20170312-12:55:51.150(-4) (percolatestudio_synced-cron.js:84) SyncedCron: Starting "scheduleNewsletter".
I20170312-12:55:51.151(-4) (percolatestudio_synced-cron.js:84) SyncedCron: Finished "scheduleNewsletter".
I20170312-12:55:51.156(-4) (percolatestudio_synced-cron.js:84) SyncedCron: Starting "scheduleNewsletter".
I20170312-12:55:51.156(-4) (percolatestudio_synced-cron.js:84) SyncedCron: Finished "scheduleNewsletter".
I20170312-12:55:51.160(-4) (percolatestudio_synced-cron.js:84) SyncedCron: Starting "scheduleNewsletter".
I20170312-12:55:51.161(-4) (percolatestudio_synced-cron.js:84) SyncedCron: Finished "scheduleNewsletter".
I20170312-12:55:51.184(-4) (percolatestudio_synced-cron.js:84) SyncedCron: Starting "scheduleNewsletter".
I20170312-12:55:51.184(-4) (percolatestudio_synced-cron.js:84) SyncedCron: Finished "scheduleNewsletter".
I20170312-12:55:51.192(-4) (percolatestudio_synced-cron.js:84) SyncedCron: Starting "scheduleNewsletter".
I20170312-12:55:51.192(-4) (percolatestudio_synced-cron.js:84) SyncedCron: Finished "scheduleNewsletter".
I20170312-12:55:51.197(-4) (percolatestudio_synced-cron.js:84) SyncedCron: Starting "scheduleNewsletter".
I20170312-12:55:51.197(-4) (percolatestudio_synced-cron.js:84) SyncedCron: Finished "scheduleNewsletter".
I20170312-12:55:51.204(-4) (percolatestudio_synced-cron.js:84) SyncedCron: Starting "scheduleNewsletter".
I20170312-12:55:51.205(-4) (percolatestudio_synced-cron.js:84) SyncedCron: Finished "scheduleNewsletter".
I20170312-12:55:51.209(-4) (percolatestudio_synced-cron.js:84) SyncedCron: Starting "scheduleNewsletter".
I20170312-12:55:51.210(-4) (percolatestudio_synced-cron.js:84) SyncedCron: Finished "scheduleNewsletter".
```

I'm on commit 81c74db42fa9a4f1cc1ff7e2fb077153df7fdb36 which is apparently Nova 1.2.

This is on Fedora 25 / x86_64 if it matters.

**Comment by SachaG** [comment]
See also #1377 and #1519. Do you have a settings file loaded? If so what do your newsletter settings look like?

**Comment by dominictracey** [comment]
This is what PR #1582 fixed. @xavcz merged it on Friday I think so it's not in 1.2.

**Comment by brendanlong** [comment]
It seems like using `devel` fixes it. I'll use that branch instead. I didn't realize `master` was a release branch.
